src/nxt_fs.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) NGINX, Inc.
+ * Copyright 2024, Alejandro Colomar <alx@kernel.org>
```

Comment:
Heh, I'm not sure the below change constitutes a _significant_ contribution...

Reply:
I guess I could move it to the next commit, but that one was already better than the previous ones, which are trivial refactors.

Reply:
Since that one already accumulates several changes to that page, I guess it's fair.

Reply:
(Although, thanks to all the refactors, the next commit is also just 2 lines in that file :D)